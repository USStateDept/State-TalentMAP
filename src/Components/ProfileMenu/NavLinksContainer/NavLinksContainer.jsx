import PropTypes from 'prop-types';

const NavLinksContainer = ({ children }) => (
  <ul>
    {children}
  </ul>
);

NavLinksContainer.propTypes = {
  children: PropTypes.node.isRequired,
};

export default NavLinksContainer;
