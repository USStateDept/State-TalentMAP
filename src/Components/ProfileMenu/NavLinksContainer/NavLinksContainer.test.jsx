import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NavLinksContainer from './NavLinksContainer';

describe('NavLinksContainerComponent', () => {
  const element = (
    <NavLinksContainer>
      <li>test 1</li>
      <li>test 2</li>
    </NavLinksContainer>
  );
  it('is defined', () => {
    const wrapper = shallow(
      element,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      element,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
