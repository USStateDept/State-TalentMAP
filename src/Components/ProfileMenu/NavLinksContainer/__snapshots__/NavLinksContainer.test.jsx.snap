// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLinksContainerComponent matches snapshot 1`] = `
<ul>
  <li>
    test 1
  </li>
  <li>
    test 2
  </li>
</ul>
`;
